README.md

```diff
@@ -277,6 +289,22 @@ podman run quay.io/redhat_msi/openshift-cli-installer \
     --cluster 'name=hyper;platform=hypershift;region=us-west-2;version=4.13.4;compute-machine-type=m5.4xlarge;replicas=6;channel-group=candidate;expiration-time=4h;timeout=1h'
 ```
 
+##### ARO cluster
+
+###### Versions
+  * Current supported versions for ARO are 4.13.23 and 4.12.25
```

Comment:
please add an explanation how to get supported versions and not hard-code them here

Reply:
done